src/locales/en/common.json

```diff
@@ -122,6 +122,7 @@
     "needToActivate": "Need to activate",
     "newSmartWalletAccountDevice": "New Smart Wallet account device",
     "notApplicable": "N/A",
+    "notEnoughGas": "Insufficient gas",
```

Comment:
```suggestion
    "notEnoughGas": "Sorry, insufficient gas was provided. Please try again or contact our support team for help",
```

Reply:
This is the button text as on this screen:

![image](https://user-images.githubusercontent.com/6368606/113698137-6bc15e80-96d4-11eb-9ec7-a0203d766eec.png)

Using longer text seems to result in odd design for a button. The design comes from Dmitry, with the reasoning for reduced text, that we assume some level of user knowledge here.

Reply:
Apologies - i thought this was a toast or something similar.